src/Analyzers/Core/Analyzers/IDEDiagnosticIds.cs

```diff
@@ -167,8 +167,6 @@ internal static class IDEDiagnosticIds
 
         public const string UseTupleSwapDiagnosticId = "IDE0180";
 
-        public const string UseParameterNullCheckingId = "IDE0190";
-
```

Comment:
Is it safe to reuse IDE0190? If not, consider adding a comment.

Reply:
I'm not sure. The safe option is definitely to just add a comment. Tagging @dotnet/roslyn-ide for guidance here.

Reply:
Since this got out into a wide release, I think a comment makes sense, lest anyone added `diagnostic.IDE0190.severity = none` to their .editorconfig file.

I heard a rumour that there might have been one or two people who didn't like the feature 😉

Reply:
Done.